extensions/dicom-microscopy/src/services/MicroscopyService.ts

```diff
@@ -69,6 +70,10 @@ export default class MicroscopyService extends PubSubService {
     });
   }
 
+  public importDicomMicroscopyViewer(): Promise<any> {
```

Comment:
This seems like a reasonable way to access the imports, regardless of whether they are dynamic or statically loaded.